### PR TITLE
Rustlang.Rust.GNU version 1.57.0

### PR DESCRIPTION
**Pull request by ykh-1992**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/40325)

### CODE REVIEW

File: manifests/r/Rustlang/Rust/GNU/1.57.0/Rustlang.Rust.GNU.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Rustlang.Rust.GNU
+PackageVersion: 1.57.0
```
Comment (ItzLevvie) [inline manifests/r/Rustlang/Rust/GNU/1.57.0/Rustlang.Rust.GNU.installer.yaml:5]:
```suggestion
PackageVersion: 1.57.0.0
```

File: manifests/r/Rustlang/Rust/GNU/1.57.0/Rustlang.Rust.GNU.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Rustlang.Rust.GNU
+PackageVersion: 1.57.0
```
Comment (ItzLevvie) [inline manifests/r/Rustlang/Rust/GNU/1.57.0/Rustlang.Rust.GNU.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 1.57.0.0
```

File: manifests/r/Rustlang/Rust/GNU/1.57.0/Rustlang.Rust.GNU.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Rustlang.Rust.GNU
+PackageVersion: 1.57.0
```
Comment (ItzLevvie) [inline manifests/r/Rustlang/Rust/GNU/1.57.0/Rustlang.Rust.GNU.yaml:5]:
```suggestion
PackageVersion: 1.57.0.0
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
image

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
You'll also need to update the version folder as well.

`manifests/r/Rustlang/Rust/GNU/1.57.0/*.yaml` to `manifests/r/Rustlang/Rust/GNU/1.57.0.0/*.yaml`

Easiest way to do this would probably be to press `.` (dot) on your keyboard on this PR which will bring up Visual Studio Code; and there you can make your changes.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/40325#pullrequestreview-847289140

**Comment by ykh-1992** [comment]
Changed parent dir name to match version number (Rustlang/Rust/GNU/1.57.0/ -> Rustlang/Rust/GNU/1.57.0.0/).

**Review by ItzLevvie — APPROVED** [review]

**Comment by ykh-1992** [comment]
Is it possible to rerun the validation process?

**Comment by OfficialEsco** [comment]
Close and reopen the PR (same button)

**Pull request merged**